utils/update-checkout

```diff
@@ -242,6 +250,11 @@ By default, updates your checkouts of Swift, SourceKit, LLDB, and SwiftPM.""")
         free-form GitHub-style comment.""",
         metavar='GITHUB-COMMENT',
         dest='github_comment')
+    parser.add_argument(
+        "--tag",
+        help="""Check out each repository to the specified tag.""",
+        metavar='TAG-NAME',
+        dest='tag')
```

Comment:
nit-pick: This `dest` parameter is unnecessary; since the argument name is `--tag` the destination will automagically be named `tag`.